index.js

```diff
@@ -10,8 +10,7 @@ module.exports = {
 
     if (type === 'head') {
       content =
-        '<script async type="text/javascript" src="https://calendly.com/assets/external/widget.js"></script>' +
-        '<link href="https://calendly.com/assets/external/widget.css" rel="stylesheet">';
+        '<script type="text/javascript" src="https://assets.calendly.com/assets/external/widget.js"></script>';
```

Comment:
What's the reasoning for this change? Removal of the `async` attribute would make downloading this script block HTML rendering